Libraries/CommonFunctions.psm1

```diff
@@ -1869,6 +1869,8 @@ Function Publish-App([string]$appName, [string]$customIP, [string]$appGitURL, [s
         Write-LogErr "$appGitURL is not set"
         return $False
     }
+    # Install dependencies
+    Install-CommonPackages -VIP $customIP -SSHPort $VMSSHPort -username $user -password $password
```

Comment:
Why create a new function that seems very specific to this purpose?
I'm thinking at calling out the utils and install_packages commands in a Run-LinuxCmd line directly.

Reply:
The new function may be used by others in future.  It's what I thought.

Reply:
@lizzha please arbitrate if this should go as a direct run-linuxcmd or keep as a new function.
If new function is to remain, then at least a rename for it is required.

I don't like it as a new function, because the list of build packages can differ, so you'll still end up installing the other deps separately.

Reply:
Update the code according to your opinion.